okonomiyaki/runtimes/runtime.py

```diff
@@ -98,7 +98,7 @@ def from_prefix_and_platform(cls, prefix, platform, version=None,
             scriptsdir = bindir = posixpath.join(prefix, "bin")
             paths = (bindir, )
 
-        if version.major == 3:
+        if version.major == 3 and platform.os_kind != OSKind.windows:
```

Comment:
I know we just want to fix the bug we have with windows at this stage but what is the rationale for using `python3` in place of `python` on Python3 ? If it is to avoid using a link to the real executable, this is not correct because the real one is `python3.5` for 3.5:

```
(py3)DP-MBP:requirements dpinte (master) $ ls -al ~/.edm/envs/py3/bin/
total 256
[...]
lrwxr-xr-x   1 dpinte  staff      7 Nov 16  2016 python -> python3
lrwxr-xr-x   1 dpinte  staff      9 Nov 16  2016 python3 -> python3.5
```